src/odemis/gui/cont/streams.py

```diff
@@ -457,6 +457,9 @@ def _on_stream_panel_destroy(self, _):
         if hasattr(self.stream, "repetition"):
             self.stream.repetition.unsubscribe(self._onStreamRep)
 
+        for entry in self.entries:
+            entry.disconnect()
+
```

Comment:
Maybe a comment here that we're disconnecting from all VAs including the standard ones.